src/Sylius/Bundle/CoreBundle/Doctrine/ORM/PaymentRepository.php

```diff
@@ -34,7 +35,7 @@ public function createListQueryBuilder(): QueryBuilder
         ;
     }
 
-    public function findOneByOrderId($paymentId, $orderId): ?PaymentInterface
+    public function findOneByOrderId(int|string $paymentId, int|string $orderId): ?PaymentInterface
```

Comment:
I wonder if it would not be better to put `mixed` for id because of possible customisations
```suggestion
    public function findOneByOrderId(mixed $paymentId, mixed $orderId): ?PaymentInterface
```
any other opinions? 🤔